content/departments/engineering/teams/growth/cloud-trial-operations.md

```diff
@@ -1,50 +1,169 @@
+1. 🟢 CE logs in, creates a password rest link
+1. 🟢 CE responds to initial SDR email, with the password reset link, and offers to help with white glove setup (AE in cc).
+1. 🟢 from there, CE-led white-glove onboarding starts: see [Second step: onboarding, and trial extend, terminate, convert](#second-step-onboarding-and-trial-extend-terminate-convert). The CE owns this instance from that point.
+
+### Second step: onboarding, and trial extend, terminate, convert
+
+1. 🟢 assigned CE sends welcome email offering to help with white glove setup (AE in cc)
+   1. Tip: Google the prospect to see if they have any OSS code or interesting projects to mention in the first outreach email.
+2. 🟢 assigned CE leads white-glove onboarding email campaigns. Exact messaging are up to the CE (with support from AE) as long as it’s helpful to the prospect. This needs to be onboarding related, helpful tips, offers to help, and check-in about commercial next steps towards the end of the trial.
+3. 🟢 From there, the assigned CE is responsible for the instance: extending, terminating, or converting the trial.
+   1. 15-days after trial start the instance should be either terminated if it's inactive, or the trial should be extended by 15 more days by raising a [trial extend request GitHub issue](../../../cloud/trial_mi.md#monitoring-trial-managed-instances). There is no need to change the license unless extending by more than 15 days. Self-serve trials inactive after 15 days are very likely to remain inactive, so we terminate them more agressively (after 15 days) than trials that come in through the sales team.
+   2. 30-days after trial start, the trial needs to be [extended, converted to paid, or terminated](../../../cloud/trial_mi.md#monitoring-trial-managed-instances)
+4. For unsuccessful trials, Malo Marrec will send feedback request email upon request by CE.
+5. Opportunities follow the same process for qualification as they do today (Stage 2 criteria). We will track “activation” in-product (Eric Brody-Moore will make sure this links to SFDC).
+
+### Email rules of engagement
 
-Here's how every team at Sourcegraph contributes to the trial workflow in the short term, and in the long term. If you want to jump to the details of the piepeline, jump to [workflow](#workflow).
+In the early stages of the trial, emails should be focused on offers for help, onboarding, tips, setting up sessions etc. After there is usage on the instance, feel free to start checking-in.
+
+### Upgrading from a random domain to a custom domain name
+
+All-prequalified leads get a pre-provisioned instance instantly. That instance has a random subdomain name (`ghjsdc.sourcegraph.com`) for technical reasons. We expect users to want to change that domain to be a vanity domain (`acme-corp.sourcegraph.com`) either when they convert, or earlier (which would be a good sign to start commercial discussions).
+
+To do so:
+
+- Request a domain name change from the cloud team by raising this issue
```

Comment:
Should we already prepare issue and link it here?

Reply:
Dammit. Caught me red-handed of not doing my homework. Yes! Will add it!

Reply:
Ooops. I committed to master.
https://github.com/sourcegraph/customer/commit/724aa1cfddc4b2f3e10de8cd4ffb6632651159fc

Reply:
@filiphaftek does that look good to you? Added a section to the cloud_trial_mi page